code_prefixes/warp2code-prefixes.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+# Processes OpenEthereum warp snapshot and collects 4-byte code prefixes of all accounts.
+#
+#    openethereum --chain=kovan snapshot --snapshot-threads=8 snapshot.warp
+#    warp2code-prefixes.py snapshot.warp
+
+import sys
+import rlp
+import snappy
+import collections
+
+prefix_map = collections.defaultdict(int)
+
+filename = sys.argv[1]
+print(f"{filename=}")
+
+with open(filename, 'rb') as f:
+	f.seek(0,2)
+	size = f.tell()
+	print(f"{size=}")
+
+	f.seek(-8,2)
+	manifest_end = f.tell()
+	manifest_off_bytes = f.read(8)
+	print(f"{manifest_off_bytes=}")
```

Comment:
Where is that format specified? Maybe add a comment with a link.

Reply:
This is Python 3.8 feature: https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging.

I don't plan to maintain this scripts. They are provided as-are if someone will to reproduce the results.

Reply:
(I meant OE manifest format)

Reply:
I had to ready OpenEthereum code for it. The documentation is either unclear or outdated. Or both.